english/marshmallow.md

```diff
@@ -10,7 +10,7 @@ Unexpectedly, the Chief Marshmallow demands to see your [references](references/
 
 *(Must be a Tuesday.)*
 
-Or a Wednesday...Thursday...maybe even a Friday.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.